src/rnp/rnpcfg.c

```diff
@@ -192,33 +205,49 @@ rnp_cfg_set(rnp_cfg_t *cfg, const char *key, const char *val)
     }
 
     cfg->vals[i] = newval;
-    return RNP_OK;
+    return true;
 }
 
-/* unset var for key, setting it to NULL if it exists in cfg */
-int
+/** @brief unset value for the key in config, making it NULL
+ *  @param cfg rnp config, must be allocated and initialized
+ *  @param key must be null-terminated string
+ *
+ *  @return true if value was found and set to NULL or false otherwise
+ **/
+bool
 rnp_cfg_unset(rnp_cfg_t *cfg, const char *key)
 {
     int i;
 
     if ((i = rnp_cfg_find(cfg, key)) >= 0) {
         free(cfg->vals[i]);
         cfg->vals[i] = NULL;
-        return RNP_OK;
+        return true;
     }
-    return RNP_FAIL;
+    return false;
 }
 
-/* set int value for the key */
-int
+/** @brief set integer value for the key in config
+ *  @param cfg rnp config, must be allocated and initialized
+ *  @param key must be null-terminated string
+ *  @param val value, which will be set
+ *
+ *  @return true if operation succeeds or false otherwise
+ **/
+bool
 rnp_cfg_setint(rnp_cfg_t *cfg, const char *key, int val)
 {
     char st[16] = {0};
     sprintf(st, "%d", val);
```

Comment:
Sure, will change it. I'm quite new to coverity, so don't know all the things yet.